server/helpers/index.js

```diff
@@ -0,0 +1,34 @@
+var jwt = require('jwt-simple');
+
+module.exports = {
+  errorLogger: function (error, req, res, next) {
```

Comment:
maybe we can move each of these helpers into their own files and require them into the index.  It would look like

```javascript
module.exports = {
errorLogger: require('./error-logger'),
errorHandler: require('./error-handler')
...
};